src/widget/form/settings/generalform.cpp

```diff
@@ -41,8 +41,6 @@ static QStringList locales = {"bg", "cs", "de", "en", "es", "fr", "hr", "hu", "i
 static QStringList langs = {"Български", "Čeština", "Deutsch", "English", "Español", "Français", "Hrvatski", "Magyar", "Italiano", "Lietuvių", "Nederlands", "Norsk Bokmål", "Polski", "Português", "Русский", "Slovenščina", "Suomi", "Svenska", "Українська", "简体中文"};
 
 static QStringList timeFormats = {"hh:mm AP", "hh:mm", "hh:mm:ss AP", "hh:mm:ss"};
-// http://doc.qt.io/qt-4.8/qdate.html#fromString
-static QStringList dateFormats = {"yyyy-MM-dd", "dd-MM-yyyy", "d-MM-yyyy", "dddd d-MM-yyyy", "dddd d-MM", "dddd dd MMMM"};
```

Comment:
I don't like this change, it removes date format which I use so, I'll add it back.